### ISSUE TITLE
Error while building Docker

### ISSUE THREAD

**Issue by Jirpm** [mxrch/GHunt#87]
```
docker build --build-arg UID=$(id -u ${USER}) --build-arg GID=$(id -g ${USER}) -t ghunt .

[+] Building 0.0s (2/2) FINISHED                                                                                                                                                                                                                              
 => ERROR [internal] load .dockerignore                                                                                                                                                                                                                  0.0s
 => => transferring context: 40B                                                                                                                                                                                                                         0.0s
 => ERROR [internal] load build definition from Dockerfile                                                                                                                                                                                               0.0s
 => => transferring dockerfile: 40B                                                                                                                                                                                                                      0.0s
------
 > [internal] load .dockerignore:
------
------
 > [internal] load build definition from Dockerfile:
------
failed to solve with frontend dockerfile.v0: failed to build LLB: error from sender: open .Trash: operation not permitted
```

**Comment by Navaxrro** [comment]
Good afternoon mate

I have interest in use this github repo to check on security issues within my team, We do photography and use google photos pretty often

I have a bit of html css and cmd experience and once I get my hands onto something I quickly learnt it.

Since it the knowledge of this security bridge I have tried to install this in my computer but I can't figure it out

I tried installing ubuntu most recent version and running some commands, installing Docker. Been looking a lot on the internet without having a good answer. Have invested in this like 10 hours with no results, between investigating and troubleshooting

Would appreciate if you, that seem knoleadgable enough cant point me in the right direction on what I should have installed or what I should be lookin for

would greatly appreciate your help 

you have a great day
